app/assets/stylesheets/layouts/starrylight.scss

```diff
@@ -1,86 +1,222 @@
+$bg_color_post_author: #42416E;
+//$bg_color_post_character_narrow: $colors_lightbrown;
+//$bg_color_post_author_narrow: $colors_brown; // these are flipped from not-narrow mode to make the quick-switcher blend in
+// TODO: Should these flip on starries too?
+$font_color_post_info: #F3F3F3;
+
+.post-character { font-weight: normal; background-color: $bg_color_post_character; }
+.post-character, .post-character a, .post-character a:visited, { color: $font_color_post_info; }
+.post-screenname { font-weight: normal; background-color: $bg_color_post_screenname; }
+.post-screenname, .post-screenname a, .post-screenname a:visited { color: $font_color_link_visited; font-style: italic; } // TODO: clean this up when messages are fixed to not use it
+.post-author { background-color: $bg_color_post_author; font-weight: normal; } // TODO: font weights different from starrydark
+.post-author, .post-author a, .post-author a:visited { color: $font_color_post_info; }
+
+$font_color_paginate_link: #F3F3F3;
+$bg_color_paginate_link_hover: #9498AC;
+$font_color_page_visited: $font_color_link_visited;
+$bg_color_page_current: #9692A8;
+$font_color_page_disabled: #58546B;
+
+.pagination .current { background-color: $bg_color_page_current; }
+.pagination a { color: $font_color_paginate_link; }
+.pagination a:visited { color: $font_color_page_visited; }
+.pagination .disabled { color: $font_color_page_disabled; }
+.pagination a:hover { background-color: $bg_color_paginate_link_hover; }
+
+/* - icons */
+//$border_color_icon_selected: #B9BBEB; // selected icon border
+$bg_color_icon: #D8D8D8;
+$bg_color_icon_credit: #D8D8D8;
+
+/*.selected-icon { border-color: #409CFF; }*/ // TODO: Why is this commented out?
+.replace-icon, .icons-box { background-color: $bg_color_icon; }
+.icon-credit { background-color: $bg_color_icon_credit; }
+
+#character-icon-selector, .time-loaded { color: #DADADA; }
+
+/* - tag items */
+$bg_color_tag_item: #968BB4;
+$font_color_tag_item: #444158;
+$font_color_tag_item_gallery: #F7F7F7;
+
+tr.gallery-tags .tag-item { background-color: $bg_color_tag_item; color: $font_color_tag_item_gallery; }
+.tag-item { color: $font_color_tag_item; }
+
+/* - link-boxes */
+// TODO: See if we can differentiate link box colors on starries
+$bg_color_linkbox_new: #6565AE;
+$bg_color_linkbox_edit: #6565AE;
+$bg_color_linkbox_favorite: #6565AE;
+$bg_color_linkbox_delete: #6565AE;
+$bg_color_linkbox_dismiss: #6565AE;
+$bg_color_linkbox_subber: #8689B7;
+
+.link-box.action-new { background-color: $bg_color_linkbox_new; }
+.link-box.action-edit { background-color: $bg_color_linkbox_edit; }
+.link-box.action-favorite { background-color: $bg_color_linkbox_favorite; }
+.link-box.action-delete { background-color: $bg_color_linkbox_delete; }
+.link-box.action-dismiss { background-color: $bg_color_linkbox_dismiss; }
+
+// TODO: I have so many questions
+.link-box { background-color: #8689B7; }
+.subber .link-box { background-color: $bg_color_linkbox_subber; }
```

Comment:
`.subber .link-box { … }` This is right – why questions?

Reply:
I think in what context those would be relevant? are there even link-boxes with no actions?

Reply:
...oh I think I made a mistake cleaning up the flash.breadcrumbs thing